defaults/main.yml

```diff
@@ -117,8 +117,10 @@ crayfish_homarus_executable_config:
       - video/mp4
       - video/x-msvideo
       - video/ogg
-    default_video: video/mp4
```

Comment:
@whikloj I was thinking that ffmpeg could be used for audio derivatives as well.  However, it not needed, can change this.  Would need to change here as well: https://github.com/Islandora-CLAW/Crayfish/blob/master/Homarus/src/app.php#L20

Reply:
@Natkeeran I think that is a good idea, but can you lay out how you were imaging it would work for defaults? 
I'm wondering if instead of defaults we should throw  415 Unsupported Media Type responses if you try to do something the service isn't configured for.